dev_guide/templates.adoc

```diff
@@ -105,13 +86,77 @@ $ osc process -f examples/sample-app/application-template-dockerbuild.json -v AD
 
 ====
 
-The JSON string can either be redirected to a file or applied directly by piping the process output to the `apply` command, as shown in the following example.
+The JSON string can either be redirected to a file or applied directly by piping the process output to the `create` command:
 
 .Redirecting JSON String
```

Comment:
i'd rather us avoid referring to it as "the json string" entirely if possible, that's pretty technical and generic.

"Applying the template output" or "Applying the template resources" or something.  whatever we're referring to the template as producing when processed.